### PR TITLE
Porfolio: Popup window

### PR DESCRIPTION
**Pull request by mateoburvi**
# Additions

In this part, we added

- JS for Project creation
- CSS for Popup Windows

Note: I had to make a new branch since my last one wasn't working properly, in this one I wont have commits from my partner, but I had some in previous branches

This was the last pull request, it shows the commits made by my coding partner
https://github.com/mateoburvi/Portfolio/pull/10

Coding partner commits

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -130,13 +130,56 @@ const projectsArray = [
...
+
+  const projectCardButton = document.createElement('button');
+  projectCardButton.classList.add('project_card_button');
+  projectCardButton.setAttribute('type', 'button');
+  projectCardButton.innerText = 'See Project';
+
+  projectCardWrapper.appendChild(projectCardButton);
+
+  projectWrapper.appendChild(projectCard);
+}
```
Comment (souad988) [inline main.js:172]: Very well done by generating the projects dynamically 👍🏼

File: main.js [hunk 2]
```diff
@@ -130,13 +130,56 @@ const projectsArray = [
...
+
+// Pop-up Window //
+
 function createPopup(index) {
```
Comment (souad988) [inline main.js:176]: Very well done the popup window shows when we click see project button 👍🏼  , I have only  two small styling issues to fix 

- [ ] reduce the padding of see live and see source buttons and make sure the spacing between title and these buttons is same as in the design reducing button padding and title size will create that space , and center the vertical alignment of title and buttons container.
- [ ] reduce the opacity of the popup container to give show the page in the background 


**your design**
Capture d’écran du 2023-07-28 21-13-36

**Figma design**
Capture d’écran du 2023-07-28 21-13-15

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @mateoburvi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good file structure
Use array of objects to store projects data
Generate projects section dynamically 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## Invalid Code Review Request ❌ 

Hi @mateoburvi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h2>You have submitted a project, that fails in linters.</h2>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @mateoburvi , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**